src/frontend/components/projects/buttons/duplicate.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import { ProjectVersionInfo } from "../../../../common/types/project-config-type";
+    import { FileSystemController } from "../../../../entities/controllers/file-system-controller";
+    import { Project } from "../../../../entities/models/project";
+    import AddItem from "../../../icons/new-icons/add-item.svelte";
+    export let version : ProjectVersionInfo;
+    export let parentProject : Project;
+
+    function duplicateVersion () : void {
+        FileSystemController.create.duplicateConfiguration(version, parentProject);
+        //TODO update after duplication
+    }
+</script>
+
+<span class="background" on:click={duplicateVersion} aria-hidden="true">
+    <span class="squares"><AddItem/></span>
```

Comment:
I don't think `<span>` should be inside another span at all. When in doubt, I usually go for divs.